front/app/containers/Admin/formBuilder/components/FormFields/index.tsx

```diff
@@ -67,72 +63,167 @@ const FormFields = ({
     watch,
     formState: { errors },
   } = useFormContext();
-  const formCustomFields: IFlatCustomField[] = watch('customFields');
   const locale = useLocale();
+  const formCustomFields: IFlatCustomField[] = watch('customFields');
+
   if (isNilOrError(locale)) {
     return null;
   }
 
+  const FormFieldsContainer = styled(Box)`
+    &:hover {
+      cursor: pointer;
+    }
+  `;
+
+  const isFieldSelected = (fieldId) => {
+    return selectedFieldId === fieldId;
+  };
+
+  const getFieldBackgroundColor = (field) => {
+    if (field.input_type === 'page') {
+      return isFieldSelected(field.id) ? colors.primary : colors.background;
+    }
+    return undefined;
+  };
+
+  const getTitleColor = (field) => {
+    if (field.input_type === 'page' && isFieldSelected(field.id)) {
+      return 'white';
+    }
+    return 'grey800';
+  };
+
+  const getIndexTitleColor = (field) => {
+    if (field.input_type === 'page' && isFieldSelected(field.id)) {
+      return 'white';
+    }
+    return 'teal300';
+  };
+
+  const getIndexForTitle = (formCustomFields, field) => {
+    if (field.input_type === 'page') {
+      // Filter out questions
```

Comment:
Very minor. Let's remove the comment here. My first interpretation was questions only but that isn't the case. It might also be my English 😄